portfolio/src/main/java/com/google/sps/data/Post.java

```diff
@@ -7,11 +7,13 @@
   private final String username;
   private final String comment;
   private final long score;
+  private final String blobKey;
 
-  public Post(String key, String username, String comment, long score) {
+  public Post(String key, String username, String comment, long score, String blobKey) {
```

Comment:
nit: add JavaDoc to public API

https://google.github.io/styleguide/javaguide.html#s7-javadoc

Reply:
would this just describe what all the private variables are used for / expected values?

Reply:
Sorry my bad - didn't realize this is a constructor. You can skip documenting in this case.